nuxeo-drive-client/nxdrive/tests/test_engine_offline.py

```diff
@@ -0,0 +1,130 @@
+'''
```

Comment:
We are adding this test case in nuxeo-drive, so we are following nuxeo-drive naming convention for file names.